index.html

```diff
@@ -50,5 +51,197 @@ <h1 class="headline-header">I'm Ismail Glade to see you!</h1>
+        </ul>
+        <button type="button" class="work-btn">See Project</button>
+      </div>
+      <div class="work-card">
+        <img
+          class="work-img"
+          src="./img/work/work-preview4.svg"
+          alt="work-preview"
+        />
+        <h3 class="work-name">Tonic</h3>
+        <div class="work-info">
+          <p class="info-main">CANOPY</p>
+          <i class="info-description info-sep fa fa-circle"></i>
+          <p class="info-description">Back End Dev</p>
+          <i class="info-description info-sep fa fa-circle"></i>
+          <p class="info-description">2015</p>
+        </div>
+        <p class="work-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="technologies">
+          <li class="lang-section">html</li>
+          <li class="lang-section">css</li>
+          <li class="lang-section">javascript</li>
+        </ul>
+        <button type="button" class="work-btn">See Project</button>
+      </div>
+    </section>
+
+    <!-- About MySelf section -->
+
+    <section class="about-me-section">
+      <div class="about-me-content">
+        <h2 class="about-me-header">About Myself</h2>
+        <p class="about-me-description">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <h2><a class="connect-link" href="#">Let's connect</a></h2>
+        <ul class="social-links">
+          <li class="social-icon">
+            <a href="#"><i class="fab fa-github-alt"></i></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"><i class="fab fa-linkedin-in"></i></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"><i class="fab fa-angellist"></i></a>
+          </li>
+          <li class="social-icon">
+            <a href="#"><i class="fab fa-twitter"></i></a>
+          </li>
+
+          <li class="social-icon">
+            <a href="#"><i class="fab fa-medium-m"></i></a>
+          </li>
+        </ul>
+        <button type="button" class="work-btn">Get my resume</button>
+
+        <div class="more-info">
+          <ul class="collapsible-menu">
+            <li class="more-coll">
+              <ul class="coll-item-list">
+                <li class="coll-tab">
+                  Languages<i class="fa fa-arrow-down"></i>
+                </li>
+
+                <li class="coll-item">
+                  <img
+                    class="coll-icon"
+                    src="./img/Icons/js.svg"
+                    alt="javascript-icon"
+                  />
+                  <p class="coll-text">JavaScript</p>
+                </li>
+                <li class="coll-item">
```

Comment:
Good job adding the missing section but it still needs a little bit of improvement. Let's go through this section step-by-step.

1. 

- [ ] The section (parent element) `<div class="more-info">` is supposed to be a `<section>` tag like you did in the other sections above it.

2. 

A list of Languages, Frameworks, and Skills. Each of them is wrapped inside a `<li>` within that list `<ul>`. this is correct right now 💯 👍 

3. 

- [ ] The Languages element is supposed to be a header for another inner (smaller) list `<ul>`. Currently, you're adding the Languages header as an element in this smaller list. Kindly work on that issue.

4.

- [ ] Kindly apply the same styling of the "Languages" header as the other elements in the same list and as the given design as well.